keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv import bounding_box
+from keras_cv.bounding_box.converters import _decode_deltas_to_boxes
+from keras_cv.layers.object_detection.anchor_generator import AnchorGenerator
+from keras_cv.layers.object_detection.roi_align import _ROIAligner
+from keras_cv.layers.object_detection.roi_generator import ROIGenerator
+from keras_cv.layers.object_detection.roi_sampler import _ROISampler
+from keras_cv.layers.object_detection.rpn_label_encoder import _RpnLabelEncoder
+from keras_cv.ops.box_matcher import ArgmaxBoxMatcher
+
+
+def _resnet50_backbone(include_rescaling=False):
```

Comment:
Is it possible to use the as_backbone() from the resnet v2 model?

Reply:
Is this merged? Can't find in the master branch that this PR is based on

Reply:
yea, that's already merged, see  https://github.com/keras-team/keras-cv/blob/7e4e7e92c87b05ccba160f868db78ddc1f44d5ef/keras_cv/models/resnet_v2.py#L384

Reply:
I also need to verify the results are identical before and after the rebase